ca/server.go

```diff
@@ -527,9 +547,11 @@ func (s *Server) signNodeCert(ctx context.Context, node *api.Node) {
 			"node.id": node.ID,
 			"method":  "(*Server).signNodeCert",
 		}).WithError(err).Errorf("failed to parse role")
-		return
+		return errors.New("failed to parse role")
 	}
 
+	s.pending[node.ID] = node
```

Comment:
I wanted this to cover the internal CA as well, for the case where there's no signing key available. If the signing key becomes available or gets unlocked later, we should retry all the pending certificates. I think this was the original idea behind implementing a reconcilation loop for the CA server, but before this PR it didn't actually work like this.